src/nnet3/nnet-diagnostics.h

```diff
@@ -97,6 +97,9 @@ class NnetComputeProb {
   // or NULL if there is no such info.
   const SimpleObjectiveInfo *GetObjective(const std::string &output_name) const;
 
+  // return string of info which summarized the objectives for different outputs.
```

Comment:
Please revert the changes to nnet-diagnostics.{h,cc}, they are no longer needed.